### ISSUE TITLE
Channel/game sorting is not working

### ISSUE THREAD

**Issue by keeepcool** [bastimeyer/livestreamer-twitch-gui#313]
*Thanks for reporting an issue!*
*Please read the contribution guidelines first!*

*Feel free to use the following template. Be as detailed as possible.*
*Don't forget to remove this text before submitting.*

----

### Description
So, I just discovered about the API change, updated to 0.15 version, created a Twitch account, and for some reason the channel and top games are not being sorted, or they are being sorted by some weird metric.
The top games that show first are games with 4-5k viewers, the top streams in the initial page range from 5k to less than 500.
...

### Expected / Actual behavior
Expect the sort system to sort by viewers
...

### Reproduction steps

1. ... Open livestreamer, press Top Games/Top channels
2. ... See games/channels ordered by viewer count 
3. ... They are sorted by some other metric/value

### Environment details (operating system, etc.)
Win 8.1 64 bits, fresh installed a couple days ago, all drivers installed, all Win updates done
...

### Comments, logs, screenshots, etc.
I have gone through all the settings but there is none related to sorting options, is this a problem on my end?

![topgames](https://cloud.githubusercontent.com/assets/9684996/18564781/08a2d2e4-7b85-11e6-8bc9-670ab0780f07.png)
![topchannels](https://cloud.githubusercontent.com/assets/9684996/18564782/08c1520a-7b85-11e6-80bc-6f7e54e9b6f5.png)

...

Best regards, thanks for the great work, and I would also help, but I only know about embedded C programming.

**Comment by bastimeyer** [comment]
Please see 
https://github.com/bastimeyer/livestreamer-twitch-gui/issues/311#issuecomment-247256038

I'll leave this issue here open, so that I don't get more dups as long as Twitch is having these issues...

**Comment by jonny2772** [comment]
I can confirm having the same issue. Did some digging, and apparently each time I open the GUI, the first time it loads without issues, it's from the second time and onward that it hides not only top streams, but also top games on each game folder. Global Top channels page seems to work as intended.
From what I saw, it seems it has issues when there's more than one page to load (from the "fetch more" separator), and the hidden games and channels might be the first page not being shown). I noticed this because, when I restarted the GUI (it shows everything the first time it loads, like I said) the first stream of the "second page" was the top one when I closed it before. Also, it doesn't seem to happen when one page is all it needs.
For instance, I opened one specific game with a single stream of 1500 viewers, it appeared without issues. Right after that I tried Fallout 4, with the same 1500 viewers total, but tons of small streams. It only showed channels with 3 or less viewers, so I suspect the top streams from the first page were hidden.
